.github/dependabot.yml

```diff
@@ -0,0 +1,8 @@
+version: 2
+updates:
+  - package-ecosystem: github-actions
+    directory: /
+    schedule:
+      interval: weekly
+    labels:
+      - no changelog
```

Comment:
I think the changes to the `.github` folder should be reverted?

Reply:
Good idea, done.